.github/workflows/update-urls.yml

```diff
@@ -0,0 +1,31 @@
+name: Update URLS
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * *'
+
+jobs:
+  updateURLS:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout ide
+        uses: actions/checkout@v3
+      - name: Checkout ide-urls
+        uses: actions/checkout@v3
+        with:
+          repository: devonfw/ide-urls
+          path: ide-urls
+          token: ${{ secrets.ACTION_PUSH_TOKEN }}
```

Comment:
This token needs repo scope permissions. If there is already a token in `devonfw/ide` for this, we can update.

Reply:
We do not have such token yet. Do you have a link to any github instructions about this?
I know how to create secrets but should I add my personal github access token there or what value should this have?

Reply:
https://docs.github.com/de/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#erstellen-eines-personal-access-token-classic

I added my own personal access token there. That was the only way I could find to make it work.

Reply:
OK, I have created such token as fine-grained org token (was more complicated than I expected) and added it as according secret to the devonfw github org (even though so far the token is only limited to ide specific repos). Lets first test this approach. Also the token requires an expiry so the github action will stop working on 2.4.2025.